File: BookStore.BusinessLogic/Views/UpdateAuthorView.cs
using BookStore.DataAccess.Models;

namespace BookStore.BusinessLogic.Views
{
    public class UpdateAuthorView
    {
        public string Id {get; set;}
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public DateTime BirthDay { get; set; }
        public DateTime? DayOfDeath { get; set; }
        public string GanreId { get; set; }
        public List<string> Ganres {get; set;}
        public List<string> Products {get; set;}
        public bool IsActive {get; set;}
    }
}

File: BookStore.Api/admin/src/shared/models/author/UpdateAuthorView.ts
import { Ganre } from "../ganre/Ganre";

export interface UpdateAuthorView {    
    id: string,
    firstName: string,
    secondName: string,
    birthDay: Date,
    dayOfDeath?: Date,
    gangeId: string,
    ganres: Ganre[],
    products: string[],
    isAvtive: boolean,
}